pytest.ini

```diff
@@ -6,3 +6,5 @@ markers =
     tags: marks tests as tags
     bs: marks tests as business services
     stakeholder: marks tests as stakeholder
+    sg: marks tests as stakeholder group
+    jf: marks tests as job function
```

Comment:
@kpunwatk please create a separate PR for adding new tests

Reply:
Done